plugins/ssh-plugin/src/ssh-plugin-backend.ts

```diff
@@ -0,0 +1,152 @@
+/*********************************************************************
+ * Copyright (c) 2018 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import * as theia from '@theia/plugin';
+import { CheService, SshKeyPair } from './common/ssh-protocol';
+import { RemoteSshKeyManager, SshKeyManager } from './node/ssh-key-manager';
+import { WsMasterHttpClient } from './node/ws-master-http-client';
+import { SshKeyServiceClient, SshKeyServiceHttpClient } from './node/ssh-key-service-client';
+import { Command } from '@theia/plugin';
+
+export async function start() {
+    const cheApi = await theia.env.getEnvVariable('CHE_API');
```

Comment:
this plug-in shouldn't use CHE_API (but use the cheAPI object provided to all che plug-ins)